introduction/hardware-options/additional-peripheral-devices/relays.md

```diff
@@ -0,0 +1,10 @@
+# Relays
+
+Relays are useful for turning higher voltage (>5V DC) auxiliary equipment on and off. 
+
+Up to three relays can be operated via joystick button functions when connected to the appropriate signal outputs.
+
+Most autopilots cannot provide power to the output rail to trigger the relays, so a 5V power supply will need to be connected to an empty output.
```

Comment:
@Williangalvani I was going to have a connection diagram for this eventually, but I don't really understand relays that much either. ArduPilot doesn't explain them very well. 

Its just using the positive and ground pins on a channel output?

Reply:
a diagram would be nice!
This is not really relay-specific. It is just that the autopilots don't provide any power on the output by default, right? This affects servos too and probably other peripherals.

I think it will be clearer if you add link to our BEC in there

Reply:
Added link to power supply. I'll need to come back to this one when I understand it a bit better.